.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
+
+services:
+- docker
+
+language: generic
+
+jobs:
+  include:
+    # Latest image build/push
+    - if:  tag IS blank OR commit_message =~ /^Triggered by upstream build/
```

Comment:
Can you give more details about the `commit_message` here. Does it comes from user? Why do we handle `Triggered by upstream build` differently?

Reply:
This message is used when gentool travis job is triggered by on of the plugin repositories.

Reply:
If I add the same message to my commit will it trigger a job?

Reply:
Yes, you will.

Reply:
It will trigger it anyway regardless of the message, because it's expected to run this job for testing PRs/commits.

Reply:
It's just to make sure versioned images are not built by plugin-triggered jobs.